types/trace-events.d.ts

```diff
@@ -0,0 +1,590 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// https://github.com/Hoten/chrome-trace-events-tsc/blob/master/dist/lh-trace-events.d.ts
+
+export namespace _TraceEvent {
+  interface Base {
+    cat: string;
+    pid: number;
+    tid: number;
+    ts: number;
```

Comment:
it would be great to have doc comments for at least these core properties (and maybe think about ways to inject more comments in trace event types we know are going to be in here, e.g. if an event has a `tdur`, print this comment before printing the `tdur: number` line, if a type has `ph: B` and is in the namespace `Whatever`, then we could print a comment saying it's the begin event for this type of trace entry, etc)